_layouts/formula-linux.html

```diff
@@ -0,0 +1,193 @@
+---
+layout: default
```

Comment:
This is the main file that I'm concerned about duplication of.

Reply:
Yeah, same. It's long and very same-y with the macOS one. I'm not sure how to make it better though as - as above - I have very little knowledge of any of the templating stuff. Does anyone who knows Jekyll more have any ideas? @EricFromCanada?

Reply:
I started over on refactoring this, and now I have something that looks good in f847a0d and there's no more `_layouts/formula-linux.html`. :tada:

Reply:
I started over on refactoring this, and now I have something that looks good in f847a0d and there's no more _layouts/formula-linux.html. 🎉